plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java

```diff
@@ -56,8 +59,9 @@ protected QueryRunner createQueryRunner()
 
     @AfterClass(alwaysRun = true)
     public final void destroy()
+            throws IOException
     {
-        kuduServer.close();
+        closeAll(kuduServer);
```

Comment:
Not strictly necessary, but helps avoid test noise if the kuduServer fails to startup and then we get a bunch of NPE